.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
I would merge nodejs and npmpublish to single build.yaml. publish needs to be conditional.

please refer to this work flow - https://github.com/dapr/java-sdk/blob/master/.github/workflows/build.yml

publish will be done only if maintainer tags v-[version]

Reply:
yes, publish would happen conditionally or it might be manual initially.